README.adoc

```diff
@@ -47,38 +47,31 @@ Learn how to...
 // Follow the intro paragraph with more details of what the user will learn, but still keep it brief compared to the overall guide.
 
 // You'll next add the various sections that are needed for a particular guide.
+* Learn to use JAX-RS and CDI
+* Learn how to build a REST service for a todo list application
 
 
 // Add this getting started section to your guide if it is applicable. Use the following include to pull in the git clone instructions from the guides-common repo.
 == Getting started
+
 include::{common-includes}/gitclone.adoc[]
 
 // This is a subsection of the "Getting started" section above. It should briefly walk the user through
 // how to setup everything in the "finish" directory and try out the finished version of what
 // they will be building.
 //=== Try what you’ll build
+
 include::{common-includes}/trywhatyoubuild-intro.adoc[]
 
+* Brief explanation on how to use the finished application and what you should expect to see
+
 // Describe what user expects to see after running the complete version of the application.
 
 include::{common-includes}/trywhatyoubuild-end.adoc[]
 
-//Write a sentence with the context like "Navigate to the `start` directory to begin." in the section where user starts working with the implementation.
```

Comment:
In general, do not remove any good instructional comments from the original template. Those will still be helpful for contributors. Perhaps rework them to fit well with the outline content.